src/hnasko_lab_to_nwb/embargo_2025/nwbconverter.py

```diff
@@ -1,28 +1,31 @@
 """Primary NWBConverter class for this dataset."""
 
 from typing import Optional
-
 from pynwb import NWBFile
-from utils import add_auditory_stimuli, add_optogenetic_stimulation, add_shock_stimuli
 
+from hnasko_lab_to_nwb.embargo_2025.utils import add_auditory_stimuli, add_optogenetic_stimulation, add_shock_stimuli
 from neuroconv import NWBConverter
 from neuroconv.datainterfaces import TDTFiberPhotometryInterface
-
+from .interfaces.demodulated_tdt_interface import DemodulatedTDTInterface
```

Comment:
If you import the interface in the init, you can simply do:
```suggestion
from interfaces import DemodulatedTDTFiberPhotometryInterface
```